x-pack/plugins/observability/public/pages/rules/components/status_context.tsx

```diff
@@ -41,22 +48,38 @@ export function StatusContext({
 
         if (status === RuleStatus.enabled) {
           await enableRule({ ...item, enabled: true });
+          if (item.muteAll) {
+            await unMuteRule({ ...item, muteAll: false });
+          }
         } else if (status === RuleStatus.disabled) {
           await disableRule({ ...item, enabled: false });
+        } else if (status === RuleStatus.snoozed) {
```

Comment:
just to make sure, we are using the internal API `rules/_find`

Reply:
@XavierM Yep we are using https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/rules.ts#L48